src/mono/wasm/runtime/library_mono.js

```diff
@@ -759,6 +759,15 @@ var MonoSupportLib = {
                 throw new Error(`Could not get a value for ${root}`);
 
             return this._resolve_member_by_name(rootObject, root, parts);
+		},
+
+		mono_wasm_set_variable_value: function (scope, index, name, newValue) {
+			console.debug (">> mono_wasm_set_variable_value " + name + " - " + newValue);
+			if (!this.mono_wasm_set_variable_value_native)
+				this.mono_wasm_set_variable_value_native = Module.cwrap ("mono_wasm_set_variable_value_native", 'number', ['number', 'number', 'string', 'string']);
```

Comment:
You can use `_register_c_fn` for this. It's meant for functions that don't return properties.

Reply:
done.